src/payment/strategies/sdk/handler/continue/continue.ts

```diff
@@ -0,0 +1,35 @@
+import { handleRedirect, isRedirect, Redirect } from './redirect';
+
+type ContinueResponse = Redirect /* | ContinueStepB | ContinueStepC */;
+
+export const isContinueResponse = (x: unknown): x is ContinueResponse => {
+    // some type checking
+    return true;
+};
+
+export const handleContinueResponse = (continueResponse: ContinueResponse): Promise<void> => {
+    // Further discern Continue sub-types
+
+    if (isRedirect(continueResponse)) {
+        return handleRedirect(continueResponse);
+    }
+
+    /*
+        Future considerations:
+
+        Some future continue steps may need to control things outside of this execution scope
+        either still inside checkout-sdk's codebase
+        or also outside in checkout-js's codebase
+
+        currently checkout-sdk doesn't know checkout-js exists (orchestration runs the other way)
+
+        for either scenario we're probably going to need some kind of pub/sub/eventing system
```

Comment:
Currently, Checkout JS can subscribe via the Checkout SDK to changes in the checkout state. That could be sufficient to start with? However, if it's related to how a payment should be processed, maybe a more specific mechanism would be more appropriate (which could still be based on the current `subscribe` mechanism).

Reply:
Yeah, I can imagine quite a bit of research is going to be needed once we come to that phase.

What I wanted to make sure with this comment is that I'm not doing anything to impede such mechanisms when we later come to add them.